packages/svelte/src/internal/client/dom/blocks/html.js

```diff
@@ -99,7 +99,7 @@ export function html(node, get_value, svg, mathml, skip_warning) {
 			// Don't use create_fragment_with_script_from_html here because that would mean script tags are executed.
 			// @html is basically `.innerHTML = ...` and that doesn't execute scripts either due to security reasons.
 			/** @type {DocumentFragment | Element} */
-			var node = create_fragment_from_html(html);
+			var node = create_fragment_from_html(html, false);
```

Comment:
can you elaborate on why we're not doing this for `{@html ...}`? It may survive hydration but it's still delivering an unexpected outcome. Feels like we should just do it in all cases

Reply:
It mainly goes down to self-closing-but-not-really tags.

```svelte
{@html "<div />"}
```
This will yield an error (I suppose if we switch to a warning it could be fine tho)